ILSANG/Sources/Views/Ranking/RankingView.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  RankingView.swift
+//  ILSANG
+//
+//  Created by Kim Andrew on 10/7/24.
+//
+
+import SwiftUI
+
+struct RankingView: View {
+    
+    @StateObject var vm = RankingViewModel(userNetwork: UserNetwork())
+    @Namespace private var namespace
+    
+    var body: some View {
+        VStack{
```

Comment:
```suggestion
        VStack{spacing:0) {
```

아래 리뷰에 padding 추가한 거랑 같이 수정 부탁드립니다!

Reply:
수정 완료